core/src/scene/drawRule.h

```diff
@@ -15,10 +15,12 @@ namespace Tangram {
 
 enum class StyleParamKey : uint8_t {
     none, order, color, width, cap, join, outline_color, outline_width, outline_cap, outline_join,
+    font_typeface, font_fill, font_stroke, font_stroke_color, font_stroke_width, font_capitalized,
+    font_visible,
 };
 
 struct StyleParam {
-    using Value = variant<none_type, std::string, Color, CapTypes, JoinTypes, int32_t, float>;
+    using Value = variant<none_type, bool, std::string, Color, CapTypes, JoinTypes, int32_t, float>;
```

Comment:
Note for future revisions: We can safely consolidate `bool`, `CapTypes`, and `JoinTypes` as `char`